CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Added
 
 - Druid DB connections can now be configured in a custom resource ([#71]).
+- Prometheus metrics enabled ([#128]).
```

Comment:
This is somewhat a breaking change for me due to the crd changes?

Reply:
marked as breaking change